### PR TITLE
docs(spring-zeebe-sdk): add repo for identity sdk

### PR DESCRIPTION
**Pull request by nicpuppa**
## Description

Spring Zeebe SDK users should add a repo into their Maven `settings.xml` file in order to make it works

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by megglos — CHANGES_REQUESTED** [review]
Thanks for getting this ready right away! One suggestion to actually recommend adding it to the projects pom, would be less invasive to the overall maven setup and portable.

File: docs/apis-tools/spring-zeebe-sdk/getting-started.md [hunk 1]
```diff
@@ -14,6 +14,22 @@ This project allows you to leverage Zeebe APIs ([gRPC](docs/apis-tools/zeebe-api
 
 ## Add the Spring Zeebe SDK to your project
 
+Add the following repository to your Maven `settings.xml`
```
[the remark this answers is not in the document]
Comment (nicpuppa) [reply docs/apis-tools/spring-zeebe-sdk/getting-started.md:17]: I wasn't aware of this, thanks

**Review by ana-vinogradova-camunda — DISMISSED** [review]
Thanks for addressing that 🚀

File: docs/apis-tools/spring-zeebe-sdk/getting-started.md [hunk 2]
```diff
@@ -14,13 +14,27 @@ This project allows you to leverage Zeebe APIs ([gRPC](docs/apis-tools/zeebe-api
...
+    <releases>
+        <enabled>true</enabled>
+    </releases>
+    <snapshots>
+        <enabled>false</enabled>
+    </snapshots>
+    <id>identity</id>
+    <name>Camunda Identity</name>
+    <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
+</repository>
```
Comment (megglos) [inline docs/apis-tools/spring-zeebe-sdk/getting-started.md:30]: 🔧 for a better UX let's also add the `<repositories>` wrapper here, so someone could just c&p the snippet into their pom.
```suggestion
<repositories>
    <repository>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>false</enabled>
        </snapshots>
        <id>identity</id>
       <name>Camunda Identity</name>
        <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
    </repository>
</repositories>
```

File: versioned_docs/version-8.5/apis-tools/spring-zeebe-sdk/getting-started.md [hunk 3]
```diff
@@ -14,7 +14,21 @@ This project allows you to leverage Zeebe APIs ([gRPC](docs/apis-tools/zeebe-api
...
+    <releases>
+        <enabled>true</enabled>
+    </releases>
+    <snapshots>
+        <enabled>false</enabled>
+    </snapshots>
+    <id>identity</id>
+    <name>Camunda Identity</name>
+    <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
+</repository>
```
Comment (megglos) [inline versioned_docs/version-8.5/apis-tools/spring-zeebe-sdk/getting-started.md:30]:
```suggestion
```suggestion
<repositories>
    <repository>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>false</enabled>
        </snapshots>
        <id>identity</id>
       <name>Camunda Identity</name>
        <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
    </repository>
</repositories>
```
```

**Review by megglos — APPROVED** [review]
LGTM! Pre-approved just one suggestion.

**Review by ana-vinogradova-camunda — APPROVED** [review]

**Pull request merged**